app/helpers/application_helper.rb

```diff
@@ -1,2 +1,20 @@
 module ApplicationHelper
+  def current_year
+    current_year = Time.now.year
+  end
+
+  def github_url(author, repo)
+    link_to "Github", "https://github.com/#{author}/#{repo}", target: "_blank"
```

Comment:
```suggestion
    link_to "Github", "https://github.com/#{author}/#{repo}", target: "_blank",  rel: 'nofollow', rel: 'noopener'
```
всякие разные есть настройки